qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -124,3 +121,18 @@
                     test_dir = '/home'
                     source_file = 'cpuid.src.tar.gz'
                     check_cpuid = './cpuid -r | grep eax=0x40000001 | uniq -c'
+                - memory_protection_key:
+                    flags = "pku ospke"
+                    variants:
+                        - @default_model:
+                            cpu_model_flags += ",+pku"
+                            HostCpuVendor.amd:
+                                no RHEL.6
+                                no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8.u0 Host_RHEL.m8.u1 Host_RHEL.m8.u2
+                        - host_model:
+                            only HostCpuVendor.amd
+                            no RHEL.6
```

Comment:
Just confirm, so it works with RHEL.7?

Reply:
@vivianQizhu yes, test pass in the fifth comment.